packages/auth/src/authClients/clerk.ts

```diff
@@ -42,6 +48,38 @@ export const clerk = (client: Clerk): AuthClientClerk => {
     ) => clerkClient(client)?.signOut(callbackOrOptions as any, options),
     signup: async (options?: SignUpProps) =>
       clerkClient(client)?.openSignUp(options || {}),
+    restoreAuthState: async () => {
+      const clerk = clerkClient(client)
+      if (!clerk) {
+        // If the client is null, we can't restore state or listen for it to happen.
+        // This behavior is somewhat undefined, which is why we instruct the user to wrap
+        // the auth provider in <ClerkLoaded> to prevent it. For now we'll just return.
```

Comment:
How about printing something to the console (in dev mode) telling the user to add `<ClerkLoaded>`?